packages/slash/src/createSlashPlugin.ts

```diff
@@ -3,20 +3,16 @@ import { createPluginFactory, removeNodes } from '@udecode/plate-common';
 import { slashOnKeyDownHandler } from './handlers/slashOnKeyDownHandler';
 import { isSelectionInSlashInput } from './queries/index';
 import { SlashPlugin } from './types';
-import { withSlash } from './withSlash';
+import { withSlashCommand } from './withSlashCommand';
 
-export const ELEMENT_SLASH = 'slash';
+export const ELEMENT_SLASH_COMMAND = 'slash_command';
```

Comment:
```suggestion
export const KEY_SLASH_COMMAND = 'slash_command';
```
since it's not an element